packages/extension-base/src/services/chain-service/index.ts

```diff
@@ -610,9 +616,43 @@ export class ChainService {
     return duplicatedSlug;
   }
 
+  private async fetchLatestData (src: string, defaultValue: unknown) {
```

Comment:
Please check it, a PR of this auto update is not ready. You should cancel this logic until everything is ok.

Reply:
Done, this was due to this PR being outdated